cfme/base/__init__.py

```diff
@@ -257,6 +278,44 @@ def advanced_settings(self):
         """"GET zones/:id/settings api endpoint to query zone configuration"""
         return self.appliance.rest_api.get(self._api_settings_url)
 
+    @property
+    def current_string(self):
+        """Returns the string ' (current)' if the appliance serving the UI request is in this zone.
+        Used to generate the appropriate tree_path for navigating configuration accordion trees."""
+        return ' (current)' if self.description == self.appliance.server.zone.description else ''
+
+    @property
+    def title_string(self):
+        """Returns a string of the form:
+
+        "Zone Description" (current)
+        [or]
+        "Zone Description"
+
+        Used to generate the title.text for Zone-related views."""
+        return f"\"{self.description}\"{self.current_string}"
```

Comment:
This looks a lot better using single quotes.

`f'"{self.description}"{self.current_string}'`

Reply:
Changed.